proto/model.proto

```diff
@@ -0,0 +1,167 @@
+// Copyright (c) 2018 Uber Technologies, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax="proto3";
+
+package jaeger.api_v2;
```

Comment:
should the `model.proto` be also in api_v2 folder? because other api_v2 files depend on it.

Reply:
It does not seem to be a problem for generating. 

I have able to generate query service in java https://github.com/jaegertracing/jaeger-analytics-java/blob/master/Makefile#L74

Reply:
I have moved it to `api_v2`, though I am not sure what implications it has